packages/core-components/src/components/ServiceCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export type ServiceCardItemType = {
+  icon: React.ReactNode;
+  title: string;
+  description: string;
+};
+
+type ServiceCardType = {
```

Comment:
should use the same convention {component}Props, or {component}Type if you want. But follow one only